packages/zoe/test/unitTests/bounty.js

```diff
@@ -0,0 +1,81 @@
+import { E } from '@agoric/eventual-send';
+import { assert, details } from '@agoric/assert';
+
+/**
+ * This contract Lets a funder endow a bounty that will pay out if an Oracle
+ * reports an event at a deadline.
```

Comment:
```suggestion
 * reports an event by a deadline.
```
?

Reply:
Ah, I take this back after seeing the code below. Hmm, maybe explain this more in case people have the same misconception as I did?

Reply:
done